setup.py

```diff
@@ -32,12 +39,17 @@ def write_git_revision():
         "suricata.update.compat.argparse",
         "suricata.update.data",
     ],
-    url="https://github.com/OISF/suricata-update",
-    license="GPLv2",
-    classifiers=[
+    "url": "https://github.com/OISF/suricata-update",
+    "license": "GPLv2",
+    "classifiers": [
         'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
     ],
-    scripts = [
+    "scripts": [
         "bin/suricata-update",
     ],
-)
+}
+
+if USE_SETUPTOOLS:
+    args["install_requires"] = ["pyyaml", ]
```

Comment:
minor: extra space at end, or is this proper style again.

Reply:
Yes. E231 on the same doc.